Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1930,6 +1930,32 @@ def get_team_members(service_url: str, team_id: str) -> list:
     return response
 
 
+def manual_get_team_members_command():
+    """
+    Retrieves and updates in context the team members of given a team_name
+    :return: List of team members
+    """
+    team_name = demisto.args().get('team_name', '')
```

Comment:
This argument should be required, so no need to make it tolerable for empty values